apps/admin-ui/components/Layout/LanguageSwapper.tsx

```diff
@@ -0,0 +1,59 @@
+import { TextField, MenuItem, Box } from "@mui/material";
+import { useLanguage } from "@easy-messe/libs/theme";
+import languageIcon from '@iconify-icons/material-symbols/language'
+import { Icon } from "@iconify/react";
+
+type CorrespondingShortLanguage = Record<string, string>;
+
+export default function LanguageSwapper() {
+    const correspondingShortLanguage: CorrespondingShortLanguage = {
+        'fr': 'Français',
+        'en': 'Anglais'
+    }
+    const supportedLanguages: string[] = ['Français', 'Anglais']
```

Comment:
change the `supportedLanguages` to an `enum`

```ts
enum supportedLanguages = {
english = 'English',
french = 'Français'
}
```

and use it like 

```ts
Object.values(supportedLanguages)
```
which will give you an array of the values that  you need.
this approach is clean but permits for extensibility in a clean way